queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIE '%mon';
```

Comment:
- [ ] I think you made a typo here. The current keyword should be `LIKE` not `LIE`. Let's work on this change.